doc/source/whatsnew/v1.2.1.rst

```diff
@@ -35,6 +35,9 @@ Fixed regressions
 - Fixed regression in comparisons between ``NaT`` and ``datetime.date`` objects incorrectly returning ``True`` (:issue:`39151`)
 - Fixed regression in :func:`pandas.testing.assert_index_equal` raising ``TypeError`` with ``check_order=False`` when :class:`Index` has mixed dtype (:issue:`39168`)
 
+We have reverted a commit that resulted in several plotting related regressions in pandas 1.2 (:issue:`38969`, :issue:`38736`, :issue:`38865`, :issue:`38947` and :issue:`39126`).
+As a result, bugs reported as fixed in pandas 1.2 related to inconsistent tick labeling in bar plots are again present (:issue:`26186` and :issue:`11465`)
```

Comment:
```suggestion
We have reverted a commit that resulted in several plotting related regressions in pandas 1.2.0 (:issue:`38969`, :issue:`38736`, :issue:`38865`, :issue:`38947` and :issue:`39126`).
As a result, bugs reported as fixed in pandas 1.2.0 related to inconsistent tick labeling in bar plots are again present (:issue:`26186` and :issue:`11465`)
```

last minor nitpick on the note: let's be explicit about the version, since it will only be in 1.2.0 (not general 1.2 series) that the regressions occurred / the other bug was temporarily fixed